content/security/for-admins/group-mapping.md

```diff
@@ -35,7 +37,32 @@ After every successful SSO sign-in authentication, the JIT provisioner performs
 
    b) If the IdP didn't provide group mappings, it checks if the user is already a member of the organization, or if the SSO connection is for multiple organizations (only at company level) and if the user is a member of any of those organizations. If the user isn't a member, it adds the user to the default team and organization configured in the SSO connection.
 
-![JIT provisioning](../images/group-mapping.png)
+![JIT provisioning](../images/jit-enabled-flow.svg)
+
+### SSO authentication with JIT provisioning disabled
+
+> **Beta feature**
+>
+> Optional Just-in-Time (JIT) provisioning is available in Private Beta when you use the Admin Console. If you're participating in this program, you have the option to turn off this default provisioning and disable JIT. This configuration is recommended if you're using SCIM to auto-provision users.
+{ .experimental }
+
+When you opt to disable JIT provisioning in your SSO connection, the following actions occur:
+
+1. Checks if there's an existing Docker account with the email address of the user that just authenticated.
+
+   a) If no account is found with the same email address, it creates a new Docker account using basic user attributes (email, name, and surname). Authentication with SSO generates a new username for this new account by using the email, name, and random numbers to make sure that all account usernames are unique in the platform.
+
+   b) If an account exists for this email address, it uses this account and updates the full name of the user’s profile if needed.
+
+2. Checks if there are any pending invitations to the SSO organization in order to auto-accept the invitation.
```

Comment:
Do we want to account for the fact that there could be multiple SSO orgs if this is a company-level SSO connection here? Or maybe that would be confusing for most customers 🤔

Reply:
I can call that out here! As long as it's clear that is just for company-level, I don't think it should be confusing 👍